h/static/scripts/app.coffee

```diff
@@ -110,6 +110,7 @@ module.exports = angular.module('h', [
 .directive('spinner', require('./directive/spinner'))
 .directive('tabbable', require('./directive/tabbable'))
 .directive('tabReveal', require('./directive/tab-reveal'))
+.directive('viaLinkDialog', require('./directive/via-dialog'))
```

Comment:
> :+1: having a little here-popup directive would be a nice refactor.

I agree - but I am only a reviewer here; I don't want to meddle in @RawKStar77's UI work ;)